### PR TITLE
[MNOE-448] Add DNS/SSL Management

### PR DESCRIPTION
**Pull request by ouranos**
Frontend PR coming soon

### CODE REVIEW

**Comment by ouranos** [comment]
See frontend screenshot at maestrano/mnoe-admin-panel#31

File: core/lib/mno_enterprise/platform_adapters/local_adapter.rb [hunk 1]
```diff
@@ -18,6 +18,16 @@ def publish_assets
         def fetch_assets
           # NOOP
         end
+
+        # @see MnoEnterprise::PlatformAdapters::Adapter#update_domain
+        def update_domain(domain_name)
+          # NOOP
```
Comment (x4d3) [inline core/lib/mno_enterprise/platform_adapters/local_adapter.rb]: isn't it to make the `update_domain` method fail in the controller ?

File: core/lib/mno_enterprise/platform_adapters/nex_adapter.rb [hunk 2]
```diff
@@ -33,8 +36,61 @@ def fetch_assets
...
 
+        # @see MnoEnterprise::PlatformAdapters::Adapter#update_domain
+        def update_domain(domain_name)
+          domain = NexClient::Domain.new(cname: domain_name)
+          domain.relationships.origin = nex_app
+          domain.save
+
+          # Display errors if any
+          if domain.errors.any?
+            # display_record_errors(domain)
```
Comment (x4d3) [inline core/lib/mno_enterprise/platform_adapters/nex_adapter.rb]: I would advocate to return the domain anyway, and display the domain.errors in the controller.

File: core/lib/mno_enterprise/platform_adapters/nex_adapter.rb [hunk 3]
```diff
@@ -33,8 +36,61 @@ def fetch_assets
...
+            false
+          else
+            domain
```
Comment (x4d3) [inline core/lib/mno_enterprise/platform_adapters/nex_adapter.rb]: not a big fan of the return false of the object pattern.

**Pull request merged**